setup.cfg

```diff
@@ -0,0 +1,21 @@
+[metadata]
+name = proposal
+url = https://github.com/tudo-astroparticlephysics/PROPOSAL
+author = PROPOSAL Developement Team
+author-email = jan.soedingrekso@tu-dortmund.de
+classifiers=
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Physics
+    Development Status :: 5 - Production/Stable
+
+license = GPLv3
+description = Monte Carlo simulation library to propagate leptons and gamma rays
```

Comment:
high energy photons or gamma photons
gamma rays sound like it can propagate through space including EBL absorption

Reply:
That's all just names for the same thing. maybe addinge "dense media" ?